app/ui-react/syndesis/src/modules/data/pages/VirtualizationViewsPage.tsx

```diff
@@ -97,253 +94,508 @@ export class VirtualizationViewsPage extends React.Component<
+                                                                  /* TD-636: Commented out for TP
+                                                                     i18nExport={t('shared:Export')} */
+                                                                  i18nPublish={t(
+                                                                    'shared:Publish'
+                                                                  )}
+                                                                  // i18nPublished={t(
+                                                                  //  'virtualization.publishedDataVirtualization'
+                                                                  // )}
+                                                                  i18nUnpublish={t(
+                                                                    'shared:Unpublish'
+                                                                  )}
+                                                                  i18nUnpublishModalMessage={t(
+                                                                    'virtualization.unpublishModalMessage',
+                                                                    {
+                                                                      name:
+                                                                        virtualization.keng__id,
+                                                                    }
+                                                                  )}
+                                                                  i18nUnpublishModalTitle={t(
+                                                                    'virtualization.unpublishModalTitle'
+                                                                  )}
+                                                                  onDelete={
+                                                                    handleDelete
+                                                                  }
+                                                                  /* TD-636: Commented out for TP
+                                                                     onExport={
+                                                                     this.handleExportVirtualization
+                                                                } */
+                                                                  onUnpublish={
+                                                                    handleUnpublish
+                                                                  }
+                                                                  onPublish={
+                                                                    handlePublish
+                                                                  }
+                                                                  // publishingLogUrl={
```

Comment:
same comment as above, remove commented out sections - unless for sure needed, the make them a TODO